### PR TITLE
Apply tolerations and nodeSelector to pods in the chart

### PR DESCRIPTION
**Pull request by TheIrritainer**
Thanks for your great app, team chatwoot. Really awesome product, really awesome also this helm chart for easy deployment.

For my use case I however need the whole release to be on a separate node and for now I'd like nothing else on that node so I'd thought of using tolerations and nodeSelector which you do specify in your readme. 

Those values however weren't propagated in the respective deployment templates so weren't working. 
I haven't check which other values were missing from the templates, this PR is merely for tolerations and nodeSelector 

also, but up to you, it might be wise to also indicate that if people want to use nodeSelector and tolerations for the dependencies that these values have to be set as well 

eg: 
redis.master.tolerations, redis.replica.tolerations, postgresql.primary.tolerations, postgresql.readReplicas.tolerations 
and these also for nodeSelector

### CODE REVIEW

**Comment by vishnu-narayanan** [comment]
@TheIrritainer Thank you for the PR. Let me get back to you on this.

**Review by vishnu-narayanan — APPROVED** [review]
@TheIrritainer Apologies for the delay in getting back. This looks good. Could you also please merge with `develop` and increment the chart version?

**Comment by TheIrritainer** [comment]
@vishnu-narayanan Sorry for the delay on my side, stopped checking the PR at one point. 

I've updated the PR with a version bump.

File: charts/chatwoot/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Overrides the image tag whose default is the chart appVersion.
 image:
   repository: chatwoot/chatwoot
-  tag: v2.1.1
+  tag: v2.1.2
```
Comment (vishnu-narayanan) [inline charts/chatwoot/values.yaml:7]:
```suggestion
  tag: v2.1.1
```

Comment (vishnu-narayanan) [reply charts/chatwoot/values.yaml:7]: Please revert this to be 2.1.1 as this is the docker image tag.

**Review by vishnu-narayanan — APPROVED** [review]
@TheIrritainer Ah no worries. Could you please resolve the comment related to the docker image version?

Also please check the test failure related to the chart version. It should now be `0.7.5`

**Comment by TheIrritainer** [comment]
> @TheIrritainer Ah no worries. Could you please resolve the comment related to the docker image version?
> 
> Also please check the test failure related to the chart version. It should now be `0.7.5`

Done!

**Comment by vishnu-narayanan** [comment]
@TheIrritainer Could you please check the failing test? It should be `0.7.6`.

**Comment by TheIrritainer** [comment]
> @TheIrritainer Could you please check the failing test? It should be `0.7.6`.

✅

**Review by vishnu-narayanan — APPROVED** [review]
LGTM

**Pull request merged**